lib/agendash.js

```diff
@@ -81,15 +81,23 @@ module.exports = function (agenda, options) {
       /*
       TODO: Need to validate user input.
       */
-      agenda.create(jobName,jobData)
-        .schedule(jobSchedule)
-        .repeatEvery(jobRepeatEvery)
-        .save(function (err) {
-          if (err) {
-            callback('Jobs not created')
-          }
-          callback()
-        });
+      var job = agenda.create(jobName,jobData)
```

Comment:
Ok for  !var instead of var === ''

But if we remove the "else if" then we won't catch the case were we need to use "job.repeatAt" instead of "job.schedule"